.gitignore

```diff
@@ -29,3 +29,5 @@ coverage.html
 common/dist/scripts/*
 
 test/spec/mocks/translations.js
+
+*.sublime*
```

Comment:
You should add this to your global gitignore instead of to the project's gitignore